### PR TITLE
Aggregate Columns - wrap commits in lambda for callbacks

### PR DESCRIPTION
**Pull request by PrimozGodec**
##### Issue

Fixes temporarily https://github.com/biolab/orange3/issues/5453

##### Description of changes
We wrap self.commit in lambda and so it is not getting called when auto-commit is not selected

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by irgolic** [comment]
You need to call the commit within the lambda
`callback=self.commit` -> `callback=lambda: self.commit()`

**Comment by PrimozGodec** [comment]
Thank you for that. It is fixed now. I don't know how it went through when I was testing 🙈

File: Orange/widgets/data/owaggregatecolumns.py [hunk 1]
```diff
@@ -53,21 +53,22 @@ def __init__(self):
...
-            callback=self.commit
+            callback=lambda: self.commit()  # pylint: disable=W0108
         )
         combo.setSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.Fixed)
 
         gui.lineEdit(
             box, self, "var_name",
             label="Variable name: ", orientation=Qt.Horizontal,
-            callback=self.commit
+            callback=lambda: self.commi()  # pylint: disable=W0108
```
Comment (markotoplak) [inline Orange/widgets/data/owaggregatecolumns.py:71]: Primož. :)

**Comment by PrimozGodec** [comment]
It should be ok now. :)

**Pull request merged**